helpers/rollback.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -e
+
+if [ -z "$TOKEN" ] || [ -z "$REPO" ]; then
+  echo "Usage: TOKEN=<Enter your CircleCI token here> REPO=<planet4-nro> ./rollback.sh"
+  exit 0
+fi
+
+user="greenpeace"
+tag="$(curl -s https://api.github.com/repos/greenpeace/${REPO}/tags | jq -r .[0].name)"
```

Comment:
Does it work without quotes around the expression `.[0].name` ? (asking because it doesn't work on cli unless I use `jq -r '.[0].name'`, I get `no maches found`

Reply:
hm.. it works here :tm: but I'll add the quotes just in case.

Reply:
Haaa my bad, it was zsh complaining, it works on bash